Dockerfile

```diff
@@ -7,14 +7,11 @@ WORKDIR /community-server
 ## Copy the package.json for audit
 COPY package*.json ./
 
```

Comment:
```suggestion

## Verify if there are known vulnerabilities in the dependencies
RUN npm audit --production --audit-level=high

```

Reply:
According to Ruben's comment, let's put the audit back: https://github.com/solid/community-server/pull/730#issuecomment-839223732

Once done, happy to merge.

Reply:
I would advice to re-enable that before the 1.0 release. Let's just merge for now.